### PR TITLE
Master

### PR DESCRIPTION
**Pull request by biomaks**
Added clearBrowserLocalStorage() and clearBrowserCookies(). Using of them will be more concise (Selenide) way than getWebDriver().manage().deleteAllCookies() and executeJavaScript("localStorage.clear()”).

### CODE REVIEW

File: src/main/java/com/codeborne/selenide/Selenide.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ private static void mockModalDialogs() {
    * Open a web page using Basic Auth credentials and create PageObject for it.
    * @return PageObject of given class
    */
-  public static <PageObjectClass> PageObjectClass open(String relativeOrAbsoluteUrl,
```
Comment (BorisOsipov) [inline src/main/java/com/codeborne/selenide/Selenide.java]: Unrelated changes

File: src/test/java/integration/ClearCookiesTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ClearCookiesTest extends IntegrationTest {
+  @Before
+  public void addCookiesBeforeTest() {
+    getWebDriver().manage().addCookie(new Cookie("username", "John Doe"));
+    Set<Cookie> cookieSet = getWebDriver().manage().getCookies();
+    assertFalse(cookieSet.isEmpty());
```
Comment (BorisOsipov) [inline src/test/java/integration/ClearCookiesTest.java]: It looks like here you test selenium webdriver feature but not selenide.
Or change to assumeFalse if you afraid that some implementations of webdriver can't add cookies in some cases

**Comment by BorisOsipov** [comment]
Hi @biomaks
Thank you for changes!
it will be better if you exclude merge commits from you pr ;)

**Comment by biomaks** [comment]
Hi Boris,
Thank you for review. May I ask someone to reject this pull request? It would be easier for me to fix and create another one.
Thanks.

**Comment by BorisOsipov** [comment]
Hi @biomaks,
You can close the pr by yourself.
